app/sitemap.ts

```diff
@@ -28,11 +29,13 @@ const sitemap = (): MetadataRoute.Sitemap => {
   const currentDate = new Date().toISOString();
 
   // This maps the URL routes into Sitemap entries
-  return [...dynamicRoutes, ...staticPaths].sort().map(route => ({
-    url: `${baseUrlAndPath}/${route}`,
-    lastModified: currentDate,
-    changeFrequency: 'always',
-  }));
+  return [...dynamicRoutes, ...staticPaths, ...EXTERNAL_LINKS_SITMAP]
+    .sort()
+    .map(route => ({
+      url: `${baseUrlAndPath}/${route}`,
```

Comment:
This will break for external links. Mapping for external links should be done separately. i.e:

```tsx
const appRoutes = [...dynamicRoutes, ...staticPaths].sort().map(route => `${baseUrlAndPath}/${route}`);

return [...appRoutes, ...EXTERNAL_LINKS_SITMAP].map(url => ({ url, lastModified: currentDate, changeFrequency: 'always' }));
```